apps/saleor-app-checkout/frontend/hooks/useGetPaymentProviderSettings.ts

```diff
@@ -12,18 +12,19 @@ export const useGetPaymentProviderSettings = <TArgs>(
   const { isAuthorized } = useAuthData();
   const { privateSettings, setPrivateSettings } = usePrivateSettings();
 
-  const domain = app?.getState().domain;
-  if (!domain) {
-    console.error(`Missing domain!`);
-  }
-  const saleorApiUrl = domain ? `https://${domain}/graphql/` : "";
+  const { app } = useAppContext();
+  const domain = app.getState().domain;
+
+  // @todo use saleorApiUrl
```

Comment:
is this still todo or done?

Reply:
Still todo:
https://github.com/saleor/saleor-app-sdk/issues/87
https://github.com/saleor/saleor/issues/10819